### PR TITLE
Adds docker image usage to README.md

### PR DESCRIPTION
**Pull request by seemethere**
Signed-off-by: Eli Uriegas <eli.uriegas@docker.com>

### CODE REVIEW

**Review by jose-bigio — APPROVED** [review]
LGTM

**Review by corbin-coleman — APPROVED** [review]
LGTM

**Review by andrewhsu — COMMENTED** [review]
LGTM

image exists:
```
$ docker pull seemethere/unir:latest
```

File: README.md [hunk 1]
```diff
@@ -13,3 +13,13 @@ changes on the Github UI and `unir` will take care of the rest!
...
 ```
+
+# Usage as a docker image!
+
+Images are up on Docker Hub!
+
+Usage is as follows:
+
+```
+docker run -d -e UNIR_WEBHOOK_SECRET -e UNIR_CLIENT_TOKEN -p 8080:8080 seemethere/unir:latest
```
Comment (jose-bigio) [inline README.md:24]: Should you add `-it` to the command? 
What about removing the extra line in the README? 

In reviewing I did see that the image was there!

**Pull request merged**